elisp.c

```diff
@@ -182,9 +183,9 @@ void set_cursor_blink(emacs_env *env, bool blink) {
                (emacs_value[]){env->make_integer(env, blink)});
 }
 
-emacs_value vterm_get_color(emacs_env *env, int index) {
+emacs_value vterm_get_color(emacs_env *env, int index, emacs_value args) {
   emacs_value idx = env->make_integer(env, index);
-  return env->funcall(env, Fvterm_get_color, 1, (emacs_value[]){idx});
+  return env->funcall(env, Fapply, 3, (emacs_value[]){ Fvterm_get_color, idx, args });
```

Comment:
Why do we need to `apply` (as opposed to funcalling the function?)

Reply:
I added new properties in `vterm--get-color` as optional `&rest args` (IMHO it looks nicer that way, since these properties are optional after all).

Here, `apply` is used to map [a list constructed in `cell_rgb_color`](https://github.com/iostapyshyn/emacs-libvterm/blob/03e335ef462500ef71c26386720043febc201626/vterm-module.c#L857) for properties onto `&rest args` in `vterm--get-color`.

Thus, it's purely cosmetic. Alternatively, `vterm--get-color` could have a signature `(defun vterm--get-color (index args) ...)` instead, without the need to use `apply` in C:

```c
return env->funcall(env, Fvterm_get_color, 2, (emacs_value[]){idx, args});
```